tests/integration/eth.test.ts

```diff
@@ -99,15 +103,21 @@ describe('Ether', async () => {
   )
 
   it('deposits ether', async () => {
-    const { ethBridger, l1Signer, l2Signer } = await testSetup()
+    const { ethBridger, l1Signer, l1Provider, l2Network, l2Signer } =
+      await testSetup()
+    const decimals = await getNativeTokenDecimals({ l1Provider, l2Network })
 
     await fundL1(l1Signer)
     const inboxAddress = ethBridger.l2Network.ethBridge.inbox
 
     const initialInboxBalance = await l1Signer.provider!.getBalance(
       inboxAddress
     )
-    const ethToDeposit = parseEther('0.0002')
+    const amount = '0.0002'
+    const ethToDeposit = scaleToNativeTokenDecimals({
+      amount: parseEther(amount),
+      decimals,
+    })
```

Comment:
Don't need to do scaling here. If ETH is used, 'parseEther' is used so `ethToDeposit` will be equal 0.0002 * 10^18 wei.  If fee token is used you can multiply amount by 10 ** decimals. Ie. if you want to deposit 0.0002 USDC `ethToDeposit` would be 0.0002 * 10 ** 6 wei. That will result with 0.0002 * 10 ** 18 wei minted on the L2.